Packs/APIVoid/Integrations/APIVoid/Pipfile

```diff
@@ -0,0 +1,33 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "==5.0.1"
+pytest-mock = "*"
+requests-mock = "*"
+pytest-asyncio = "*"
+pytest-xdist = "*"
+pytest-datadir-ng = "*"
+freezegun = "*"
+flake8 = "*"
+
+[packages]
+certifi = "==2019.6.16"
+chardet = "==3.0.4"
+dateparser = "==0.7.1"
+idna = "==2.8"
+olefile = "==0.46"
+python-dateutil = "==2.8.0"
+pytz = "==2019.1"
+regex = "==2019.6.8"
+requests = "==2.22.0"
+six = "==1.12.0"
+tzlocal = "==1.5.1"
+urllib3 = "==1.25.3"
+PyYAML = "==5.1.1"
```

Comment:
do we need all of them?

Reply:
it's all the packages in the base docker